ledger/voters_test.go

```diff
@@ -261,9 +259,8 @@ func TestTopNAccountsThatHaveNoMssKeys(t *testing.T) {
 	defer ml.Close()
 
 	conf := config.GetDefaultLocal()
-	au, ao := newAcctUpdates(t, ml, conf)
-	defer au.close()
-	defer ao.close()
+	_, ao := newAcctUpdates(t, ml, conf)
+	// au and ao are closed via ml.Close() -> ml.trackers.close()
```

Comment:
`au` is no longer a variable.
```suggestion
	// accountUpdates and onlineAccounts are closed via: ml.Close() -> ml.trackers.close()
```